documentation/public/user-guide/npm-package/using-the-cursor.md

```diff
@@ -0,0 +1,47 @@
+# Using the Cursor
+
```

Comment:
IMO we should keep an intro here about `goTo*` being the main API here and/or back-link to Cursor in the Concepts section for more info.

Sub-tree navigation and node accessors are right to give a brief intro/explanation and I think we should give an intro here as well.

I understand the urge to distill the code content here but, after all, it's not a very specific how-to guide like "How to install Docker in WSL on Microsoft Windows Server 2022" with concrete and specific steps but rather is *the* guide on Cursors and how to use them, so an intro would fit.

Reply:
Added a link to the specific concept that explains this in detail :+1